I'm seeing this new static check error even though I did not make any changes to this file
```
Run mypy for core......................................................................Failed
- hook id: run-mypy
- exit code: 1

airflow/sensors/base.py:106: error: Name "Iterable" is not defined 
[name-defined]
        valid_modes = ["poke", "reschedule"]  # type: Iterable[str]
        ^
airflow/sensors/base.py:106: note: Did you forget to import it from "typing"? (Suggestion: "from typing import Iterable")
Found 1 error in 1 file (checked 1022 source files)
```

--- a/airflow/api_connexion/endpoints/health_endpoint.py
+++ b/airflow/api_connexion/endpoints/health_endpoint.py
@@ -41,7 +41,6 @@ def get_health() -> APIResponse:
                 scheduler_status = HEALTHY
     except Exception:
         metadatabase_status = UNHEALTHY
- 
     try:
         triggerer_job = TriggererJob.most_recent_job()
 

--- a/airflow/api_connexion/endpoints/health_endpoint.py
+++ b/airflow/api_connexion/endpoints/health_endpoint.py
@@ -16,8 +16,6 @@
 # under the License.
 from __future__ import annotations
 
-from typing import Optional
-
 from airflow.api_connexion.schemas.health_schema import health_schema
 from airflow.api_connexion.types import APIResponse
 from airflow.jobs.scheduler_job import SchedulerJob
@@ -33,7 +31,7 @@ def get_health() -> APIResponse:
     latest_scheduler_heartbeat = None
     latest_triggerer_heartbeat = None
     scheduler_status = UNHEALTHY
-    triggerer_status: Optional[str] = UNHEALTHY
+    triggerer_status: str | None = UNHEALTHY
     try:
         scheduler_job = SchedulerJob.most_recent_job()
 

--- a/airflow/api_connexion/endpoints/health_endpoint.py
+++ b/airflow/api_connexion/endpoints/health_endpoint.py
@@ -16,6 +16,8 @@
 # under the License.
 from __future__ import annotations
 
+from typing import Optional
+
 from airflow.api_connexion.schemas.health_schema import health_schema
 from airflow.api_connexion.types import APIResponse
 from airflow.jobs.scheduler_job import SchedulerJob
@@ -31,7 +33,7 @@ def get_health() -> APIResponse:
     latest_scheduler_heartbeat = None
     latest_triggerer_heartbeat = None
     scheduler_status = UNHEALTHY
-    triggerer_status = UNHEALTHY
+    triggerer_status: Optional[str] = UNHEALTHY
     try:
         scheduler_job = SchedulerJob.most_recent_job()
 

